app/src/androidTest/java/com/tpp/theperiodpurse/NavigationOnboardTest.kt

```diff
@@ -0,0 +1,162 @@
+package com.tpp.theperiodpurse
+
+import android.app.Application
+import androidx.activity.ComponentActivity
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.test.*
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.navigation.compose.ComposeNavigator
+import androidx.navigation.testing.TestNavHostController
+import com.tpp.theperiodpurse.ui.onboarding.OnboardViewModel
+import dagger.hilt.android.testing.HiltAndroidRule
+import dagger.hilt.android.testing.HiltAndroidTest
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import com.tpp.theperiodpurse.MainActivity
+import java.text.DateFormat
+import java.util.*
+import javax.inject.Inject
+
+@HiltAndroidTest
+class NavigationOnboardTest {
+    @get:Rule
+    val composeTestRule = createAndroidComposeRule<ComponentActivity>()
+    private lateinit var navController: TestNavHostController
+
+    @Inject
+    lateinit var onboardViewModel: OnboardViewModel
+
+    @get:Rule
+    // Used to manage the components' state and is used to perform injection on tests
+    var hiltRule = HiltAndroidRule(this)
+
+    @Before
+    fun setupNavHost() {
+        hiltRule.inject()
+        composeTestRule.setContent {
+            navController =
+                TestNavHostController(LocalContext.current)
+            navController.navigatorProvider.addNavigator(
+                ComposeNavigator()
+            )
+            ScreenApp(navController = navController, skipOnboarding = false, skipWelcome = false, viewModel =
+            onboardViewModel) { signIn() }
+        }
+    }
+
+    @Test
+    fun appNavHost_Onboard_navigatesToSummary_viewModel() {
+
+        // welcome page
+        navController.assertCurrentRouteName(OnboardingScreen.Welcome.name)
+
+
+        // next to quesiton one page
+        composeTestRule.onNodeWithContentDescription("Next").performClick()
+            .performClick()
+        navController.assertCurrentRouteName(OnboardingScreen.QuestionOne.name)
+
+        // text field input test and next to question two page
+        val textFieldValue = "5"
+        composeTestRule.onNodeWithContentDescription("Pick Days")
+            .performTextInput(textFieldValue)
+        composeTestRule.onNodeWithContentDescription("Pick Days")
```

Comment:
As with one of the comments above, if i don't input anything in the textFieldValue and click "done", the app crashes. Can you add test for that?